src/share/export.js

```diff
@@ -0,0 +1,48 @@
+import fs from '../lib/fs';
+const sh = new fs.Shell();
+import WebTorrent from 'webtorrent';
+
+const files = [];
+
+document.getElementById('btnSeed').addEventListener('click', startSeeding);
```

Comment:
Let's drop the `btnSeed` completely, and do this in `window.onload = () => {...}` so that it happens as soon as you open the page vs. waiting on the user to initiate.

Then you can move `const files = []` into the body of that function, as well as your other 2 functions:

```js
window.onload = () => {
  const files = [];
  const processPath = (path, next) => {
    ..
  }
  // move body of startSeeding (i.e., drop the function) here...
};
```